src/plays/dev-jokes/devJokes.css

```diff
@@ -0,0 +1,76 @@
+.maincontanier {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  background-image: url("background.jpg");
+  background-position: center;
+  background-size: cover;
+  height: 100%;
+  width: 100%;
+}
+.jokecontanier {
+  margin: 20px;
+  padding: 20px;
+  width: 600px;
+  min-height: 400px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 10px;
+  box-shadow: 0px 6px 10px rgba(0, 0, 0, 0.25);
+  transition: all 0.2s;
+  margin-top: 30px;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(199, 183, 183, 0.4);
```

Comment:
repeated property.

Reply:
This is basically for creating a blur  type effect..

Reply:
It will just overwrite the previous value with the later one. Thats how CSS specificity and order works.
refer 
![image](https://user-images.githubusercontent.com/7145967/181313784-50529ba2-31fe-4013-8d10-211662112e2c.png)

Reply:
repaired it